charts/tfy-karpenter-config/values.yaml

```diff
@@ -222,3 +222,40 @@ karpenter:
       else
           echo "CONTAINERD_VERSION is empty or not within the specified range."
       fi
+
+  ## provisioner for inferentia workloads
+  inferentiaProvisionerSpec:
+    ## @param karpenter.inferentiaProvisionerSpec.enabled Enable inferentia provisioner for inferentia nodes
+    ##
+    enabled: false
+
+    ## @param karpenter.inferentiaProvisionerSpec.consolidation.enabled Enable consolidation for inferentia provisioner
+    ##
+    consolidation:
+      enabled: false
```

Comment:
Any reason to keep this `false` by default ?

Reply:
Yes. It is not tested yet. I could not bring up larger machines having multiple accelerators due to quota unavailability.